docs/html/_sources/Documentation/3_DesignDecisions/3_Database/1_DatabaseDesign.rst.txt

```diff
@@ -0,0 +1,36 @@
+Designing the Database
+**************************
+
+This is a log of major design decisions and reasoning made during creation of the database section of decompy.
+At the time of writing this, this is made up of one class based off of a design diagram, the RepoFiltering class, and the RepoStructure class.
```

Comment:
Mentioning here in the rst as well since that is what should be changed.... this seems incorrect and too liable to change in the next short while to bother mentioning. I know it is based off of mine, but the Repo filtering alone likely wont change soon. Also database design is rather independent of the data gathering, so why mention it here